recipe/meta.yaml

```diff
@@ -1,52 +1,49 @@
-{% set name = "spyder" %}
 {% set version = "3.3.1" %}
-{% set sha256 = "992d74f07961df9a9b9aa495fa2d87832699ada5323850dc3e26ac0ff640b170" %}
 
 package:
-  name: {{ name|lower }}
+  name: spyder
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/s/spyder/spyder-{{ version }}.tar.gz
+  sha256: 992d74f07961df9a9b9aa495fa2d87832699ada5323850dc3e26ac0ff640b170
   patches:
-    # Prevent a crash in macOS with Python 3
-    - zmq.patch
+    - 0001-Hack-import-zmq-in-start.py.patch
 
 build:
-  number: 2
+  number: 3
   entry_points:
     - spyder = spyder.app.start:main
   osx_is_app: True
 
 requirements:
-  build:
+  host:
     - python
+    - pip
     - sphinx
   run:
     - python
-    - pyqt =5
-    - qtpy >=1.2.0
-    - qtawesome >=0.4.1
-    - rope >=0.9.4
-    - pyflakes
+    - chardet >=2.0
+    - cloudpickle
     - jedi >=0.9
-    - qtconsole >=4.2
-    - pickleshare
+    - keyring  # [not (linux and py2k)]
     - nbconvert
-    - pygments >=2.0
-    - sphinx
-    - pycodestyle
+    - numpydoc
+    - pickleshare
     - psutil
+    - pycodestyle
+    - pyflakes
+    - pygments >=2.0
     - pylint
-    - pyzmq
-    - chardet >=2.0
-    - numpydoc
-    - cloudpickle
-    - keyring  # [not (linux and py2k)]
-    - spyder-kernels =0
+    - pyqt >=5.6,<6
```

Comment:
This is dangerous. The previous pin was set to avoid compatibility issues with `defaults`.

Reply:
You can still install PyQt 5.6 from defaults, so how is that mix avoided?

Reply:
Not b/c 5.6 from defaults is OK.